conf/iptables-rules.v4

```diff
@@ -0,0 +1,14 @@
+*filter
+:INPUT ACCEPT [100:6731]
```

Comment:
Is there a need to be so permissive? I'd rather see just the important ports open. Additionally, is it necessary to open 27017 outside the LAN?